entry_tasks/movie_picker/movie_picker.py

```diff
@@ -0,0 +1,66 @@
+if __name__ == "__main__":
+    GENRES = {
+        'comedy': ['Meet the Parents', 'Anger Management'],
+        'adventures': ['Mummy'],
+        'romantic': ['Vanilla Sky', 'Meet Joe Black'],
+        'drama': ['Meet Joe Black'],
+        'thriller': ['Vanilla Sky'],
+        'action': ['Mission Impossible']
+    }
+
+    ACTORS = {
+        'Robert De Niro': ['Meet the Parents'],
+        'Ben Stiller': ['Meet the Parents'],
+        'Adam Sandler': ['Anger Management'],
+        'Jack Nicholson': ['Anger Management'],
+        'Brendan Fraser': ['Mummy'],
+        'Rachel Weisz': ['Mummy'],
+        'Tom Cruise': ['Vanilla Sky', 'Mission Impossible'],
+        'Penelope Cruz': ['Vanilla Sky'],
+        'Cameron Diaz': ['Vanilla Sky'],
+        'Brad Pitt': ['Meet Joe Black'],
+        'Anthony Hopkins': ['Meet Joe Black'],
+        'Jeremy Renner': ['Mission Impossible']
+    }
+
+    def search_by_genre():
+        all_genres = ", ".join(list(GENRES.keys()))
+        print(f"{'Available Genres: '} {all_genres}")
+        user_choose_gerne = input("Enter genre: ").strip()
+        if user_choose_gerne in GENRES:
+            movies_list = GENRES[user_choose_gerne]
+            print(f'Available movies: {movies_list}')
+            user_choose_film = input("Enter movie: ").strip()
+            if user_choose_film in GENRES[user_choose_gerne]:
+                print(f'Movie to watch: {user_choose_film}. Genre: {user_choose_gerne}.')
+            else:
+                print('Incorrect film name! Try again')
+        else:
+            print('Incorrect gerne! Try again')
+
+    def search_by_actor():
```

Comment:
Seems that search_by_genre and search_by_actor differ by few words only ?
Maybe we can do something with it? :) since it's kinda duplication, right?

Reply:
@mykytapavlov done